Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -213,6 +213,13 @@ public extension Tensor {
     func squeezingShape(at axes: [Int]) -> Tensor {
         return Raw.squeeze(self, squeezeDims: axes.map(Int32.init))
     }
+
+    /// Returns a `Tensor` by clipping tensor values to a specified min and max.
+    @inlinable
+    @differentiable(wrt: self where Scalar: TensorFlowFloatingPoint)
+    func clipped<T>(min minimum: Tensor<T>, max maximum: Tensor<T>) -> Tensor{
```

Comment:
This won't work because you defined `clipped(min:max:)` in a general extension where `Scalar` is not constrained as it should be. `Raw.clipByValue(t:clipValueMin:clipValueMax:)` is defined as the following:
```swift
public static func clipByValue<T: Numeric & TensorFlowScalar>(
    t: Tensor<T>,
    clipValueMin: Tensor<T>,
    clipValueMax: Tensor<T>
) -> Tensor<T>
```
Namely, `T` is constrained to `Numeric` in addition to `TensorFlowScalar`.  Therefore, you should define method `clipped(min:max:)` in an extension that constrains `Scalar` to `Numeric`.  And the `clipped(min:max:)` method itself should not have any generic parameter (i.e. no `<T>`) because its genericity comes from the extension.

```swift
public extension Tensor where Scalar: Numeric {
    func clipped(min minimum: Tensor, max maximum: Tensor) -> Tensor {
        ....
    }
}
```